components/raftstore/src/store/snap/io.rs

```diff
@@ -39,7 +39,9 @@ pub trait StaleDetector {
 #[derive(Clone, Copy, Default)]
 pub struct BuildStatistics {
     pub key_count: usize,
-    pub total_size: usize,
+    pub total_kv_size: usize,
```

Comment:
Pls add some necessary annotations of the three different objects.

Reply:
+1, having some descriptions will be helpful. Is `total_kv_size` the uncompressed size and `total_sst_size` the compressed size?

Reply:
done

Reply:
> +1, having some descriptions will be helpful. Is `total_kv_size` the uncompressed size and `total_sst_size` the compressed size?

yes.